packages/picasso.js/src/core/chart-components/axis/axis-node-builder.js

```diff
@@ -214,18 +242,30 @@ export default function nodeBuilder(isDiscrete) {
     const layered = state.labels.activeMode === 'layered';
     let majorTickNodes;
 
+    if (settings.arc.isRadial) {
```

Comment:
What happens if the settings does not have arc defined. The old versions does not have arc defined in settings.